lib/Transforms/Scalar/DxilLoopUnroll.cpp

```diff
@@ -137,11 +137,13 @@ class DxilLoopUnroll : public LoopPass {
     GetPassOptionUnsigned(O, "MaxIterationAttempt", &MaxIterationAttempt,
                           false);
     GetPassOptionBool(O, "OnlyWarnOnFail", &OnlyWarnOnFail, false);
+    GetPassOptionBool(O, "StructurizeLoopExits", &StructurizeLoopExits, false);
```

Comment:
I don't understand why this change is here. I don't see any code reading this option. Is it needed for the fix?

Reply:
By default, we don't structurize loop exit, and it's controlled by an -opt-enable, which translates to an argument to DxilLoopUnroll pass. When running dxopt.exe, there was no way to turn on the structurizer, so I added this option here. Plus the option should exist anyway for the argument. Without it, our -Odump is actually wrong.

Reply:
Oops, I just straight up forgot to add it to the test.

Reply:
Fixed now.